core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/verticalscale/CoreVerticalScaleState.java

```diff
@@ -6,8 +6,11 @@
 
 public enum CoreVerticalScaleState implements FlowState {
     INIT_STATE,
+    STACK_PREPARATION_STATE,
     STACK_VERTICALSCALE_FAILED_STATE,
 
+    STACK_VERTICALSCALE_PREPARATION_FAILURE_EVENT,
+
```

Comment:
This is required - because CoreVerticalScaleResult is used by a different flow step and CoreVerticalScalePreparationResult. I had thought of using the same failure event - but when I did the same in a previous flow change, I was asked to create a new class by Duckies even if the classes are identical

Reply:
This is a `State` enum. There is `CoreVerticalScaleEvent.STACK_VERTICALSCALE_PREPARATION_FAILURE_EVENT` that contains the failure selector.

Reply:
removed